src/impl_methods.rs

```diff
@@ -2606,14 +2618,14 @@ where
             };
             // Delegate to mapv_into() using the wrapped closure.
             // Convert output to a uniquely owned array of type Array<A, D>.
-            let output = self.mapv_into(f).into_owned();
+            let output = self.mapv_into(f);
             // Change the return type from Array<A, D> to Array<B, D>.
```

Comment:
The implementation do not seem to match the types after this change?

Reply:
You are write, I forgot to update the comments.  We are no longer using a concrete `Array` as the return type, so the `into_owned()` is no longer needed.  Fixing the comments with a force push.